bitnami/redis-cluster/7.0/debian-11/prebuildfs/opt/bitnami/scripts/libfile.sh

```diff
@@ -34,7 +34,7 @@ replace_in_file() {
     if [[ $posix_regex = true ]]; then
         result="$(sed -E "s${del}${match_regex}${del}${substitute_regex}${del}g" "$filename")"
     else
-        result="$(sed "s${del}${match_regex}${del}${substitute_regex}${del}g" "$filename")"
+            result="$(sed "s${del}${match_regex}${del}${substitute_regex}${del}g" "$filename")"
```

Comment:
Just to follow the structure and the ident.
```suggestion
        result="$(sed "s${del}${match_regex}${del}${substitute_regex}${del}g" "$filename")"
```